packages/icons/src/PentahoIcon.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  FontAwesomeIcon,
+  FontAwesomeIconProps,
+} from "@fortawesome/react-fontawesome";
+import { getColor, HvColorAny } from "@hitachivantara/uikit-styles";
+
+import { IconBaseProps, StyledIconBase } from "./IconBase";
+import { getSizeStyles, getSvgSize } from "./utils";
+
+export interface PentahoIconProps
+  extends Omit<FontAwesomeIconProps, "color" | "size"> {
+  /** The size of the SVG icon. Takes in a `number` in pixels or any `HvSize` or `IconSize`.
+   *
+   * @example
+   * size={16} // 16px
+   * size="S" // 16px
+   * size="sm" // 16px
+   * size="md" // 32px
+   *
+   * @default "S"
+   */
+  size?: IconBaseProps["size"];
+  /**
+   * A color to override the default icon colors.
+   * Accepts any valid CSS color or color from the UI Kit palette.
+   * @example "brand"
+   */
+  color?: HvColorAny;
+  /** Gives the icon padding.  */
+  container?: boolean;
+}
+
+/**
+ * A wrapper for the [**react-fontawesome**](https://github.com/FortAwesome/react-fontawesome) component
+ * to easily apply our theme sizes colors and margins to the `font-awesome` icons.
+ */
+export const PentahoIcon = ({
```

Comment:
I added Pentaho because I don´t think that is supposed to be mixed usage of icons, users using Next DS should use the current icons and Pentaho users should use the `font-awsome` icons. `HvExternalIcon` seems to general since this currently only works for `font-awsome`. I could change it to `HvFontAwesomeIcon` as that is the name of the react component it wraps, but it seems strange to me.

Reply:
Okay, got it. If the rest of the team is okay with the name, I don't mind. I thought this component could be used by anyone regardless of the theme. The `Hv` prefix is just missing though. We should add it.